app/models/footnote.rb

```diff
@@ -13,14 +13,13 @@ class Footnote < Sequel::Model
                                                      footnote_id],
                                        right_primary_key: %i[footnote_description_period_sid
                                                              footnote_type_id
-                                                             footnote_id],
-                                       eager_block: (->(ds) { ds }) do |ds|
```

Comment:
When `reload: true` basically turns off eager loading and the corresponding eager_block. When we removed the reload we ended up calling the eager_block instead of the relation block on line 18/19 below. This was breaking some tests so this change fixes those expectations.

Reply:
I don't know why we'd want to execute something different when eager loading (for example here we were skipping the TimeMachine and the ordering functionality.

Reply:
Probably worth a review

Reply:
I suspect this might account for a decent number of the forced reloads/presumed TimeMachine issues